sdk/core/Azure.Core/src/Shared/DiagnosticScope.cs

```diff
@@ -293,7 +293,7 @@ public void AddLink(string traceparent, string tracestate, IDictionary<string, s
 
                     if (_startTime != default)
                     {
-                        _currentActivity.SetStartTime(_startTime.DateTime);
+                        _currentActivity.SetStartTime(_startTime.UtcDateTime);
```

Comment:
Can we use the approach from https://github.com/Azure/azure-sdk-for-net/pull/26872

Reply:
Thanks for the pointer. No, I don't think that approach would fix it.
Note that _startTime is a DateTimeOffset
DateTimeOffset.DateTime returns a DateTime with of Unspecified kind.
We want UTC, so we should use UtcDateTime.

I spot checked the callers of DiagnosticScope.SetStartTime and they're already using UTC, so converting to UTC `.ToUniversalTime()` would be a no-op, and it doesn't fix the issue.

Reply:
Ah, I see your point. Regardless of how the `_startTime` DateTimeOffset is constructed, the DateTime property will always give back an Unspecified DateTime rather than a UTC DateTime (even though the value is actually UTC).

Reply:
Exactly.